source/community/release_notes/redwood.rst

```diff
@@ -39,3 +241,5 @@ Researcher & Data Experiences
 
```

Comment:
for Researcher & Data, we should add a link to the product notes about the Aspects release, and detail what it takes to install it on your instance.

Reply:
OK please forgive my ignorance I have no idea what that means :) Who should I talk to to get that excellent suggestion transmuted into text I can include?

Reply:
@bmtcril  what should the developer & operator release notes include about Aspects (eg how to install/enable/administer)

Reply:
FWIW I *THINK* this is the only outstanding change we need to make so we can be ready for tomorrow's release. Thanks for the feedback we've gotten thus far all!